src/hotspot/share/utilities/accessFlags.hpp

```diff
@@ -71,14 +62,14 @@ class AccessFlags {
   bool is_synthetic   () const         { return (_flags & JVM_ACC_SYNTHETIC   ) != 0; }
 
   // get .class file flags
-  jint get_flags               () const { return (_flags & JVM_ACC_WRITTEN_FLAGS); }
+  u2 as_int           () const         { return _flags; }
```

Comment:
Why do you call this `as_int` when it returns a u2? This is just confusing as per numerous other comments. It would be clearer to just have `u2 get_flags()` and do away with `as_int()` and `as_short()`.